util/lambda_trigger/lambda_trigger_test.go

```diff
@@ -10,8 +10,8 @@ import (
 const test_oss_product_formula = "nomad"
 const test_oss_product_cask = "boundary-desktop"
 const test_ent_product_formula = "nomad-enterprise"
-const test_oss_version_formula = "1.3.5"
```

Comment:
Now I know why the tests weren't run in CI before 😆  they will fail anytime a version bump happens because this is a static value. I'll remove these from running in CI tomorrow.